src/Microsoft.Graph.Cli.Core.Tests/Http/UriReplacement/MeUriReplacementTests.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using Microsoft.Graph.Cli.Core.Http.UriReplacement;
+using Xunit;
+
+namespace Microsoft.Graph.Cli.Core.Tests.Http.UriReplacement;
+
+public class MeUriReplacementTests
+{
+    [Fact]
+    public void Returns_Null_When_Given_A_Null_Url()
+    {
+        var replacement = new MeUriReplacement();
+
+        Assert.Null(replacement.Replace(null));
+    }
+
+    [Fact]
+    public void Returns_Original_Uri_When_No_Match_Is_Found()
+    {
+        var uri = new Uri("http://example.com/test");
+        var replacement = new MeUriReplacement();
+
+        Assert.Equal(uri, replacement.Replace(uri));
+
+        uri = new Uri("http://example.com/users/messages");
+        Assert.Equal(uri, replacement.Replace(uri));
+
+        uri = new Uri("http://example.com/v1.0/users/messages");
+        Assert.Equal(uri, replacement.Replace(uri));
+
+        uri = new Uri("http://example.com/users/test/me");
+        Assert.Equal(uri, replacement.Replace(uri));
+
+        uri = new Uri("http://example.com/a/b/users/test/me");
+        Assert.Equal(uri, replacement.Replace(uri));
+    }
```

Comment:
nit: Use theory instead.
```suggestion
    [Theory]
    [InlineData("http://example.com/test")]
    [InlineData("http://example.com/users/messages")]
    [InlineData("http://example.com/v1.0/users/messages")]
    [InlineData("http://example.com/users/test/me")]
    [InlineData("http://example.com/a/b/users/test/me")]
    public void Returns_Original_Uri_When_No_Match_Is_Found(string inputUri)
    {
        var uri = new Uri(inputUri);
        var replacement = new MeUriReplacement();

        Assert.Equal(uri, replacement.Replace(uri));
    }
```